palette/src/lch.rs

```diff
@@ -136,6 +136,18 @@ where
         T::zero()
     }
 
+    /// Return the maximum `chroma` value that is representable for
+    /// the given lightness and hue values.
+    pub fn max_chroma_for_lh(&self) -> T {
```

Comment:
Again, the bounds are for l and h defined in CIELch(uv), which is different from the CIELch(ab) of this file.